src/main/java/org/openrewrite/kotlin/internal/KotlinPrinter.java

```diff
@@ -858,10 +858,6 @@ public J visitTernary(J.Ternary ternary, PrintOutputCapture<P> p) {
         @Override
         public J visitTypeCast(J.TypeCast typeCast, PrintOutputCapture<P> p) {
             beforeSyntax(typeCast, Space.Location.TYPE_CAST_PREFIX, p);
-            boolean printParens = !typeCast.getMarkers().findFirst(OmitParentheses.class).isPresent();
-            if (printParens) {
-                p.append('(');
-            }
```

Comment:
How does this not break kotlin code that uses lambdas?
```kotlin
listOf(1,2,3).forEach { println(it) } // very common to omit parentheses here
```

Reply:
I am not sure I understand this question, 
I think for lambda, it's handled by `visitLambda` method in this file.

Reply:
Oh yeah this is type casts.
In kotlin there are two kinds of type casts, the java-style `(Foo) bar` and `bar as Foo`. Does the latter still print correctly with this change?

Reply:
Yeah, this code is actually removed from @traceyyoshima  initial change, he might have more context about why remove this, but I verified the printer works for the latter for the below case. 
```kotlin
fun main() {
    val obj: Any = "Hello"
    val str: String = obj as String
}
```
but I am not sure how to do java-style type case in Kotlin.
I tried this but doesn't work
```kotlin
fun main() {
    val obj: Any = "Hello"
    // val str: String = obj as String
    val str2: String = (String) obj
}
```